src/config/config.ts

```diff
@@ -1,8 +1,33 @@
+import { type IToken, type IWalletOption } from '@/types/common';
+
 const contractName = process.env.REACT_APP_CONTRACT_ID ?? 'v2.keypom.testnet';
 const cloudflareIfps = process.env.REACT_APP_CLOUDFLARE_IFPS ?? 'https://cloudflare-ipfs.com/ipfs';
 // eslint-disable-next-line no-console
 console.log(process.env.REACT_APP_NETWORK_ID, process.env.REACT_APP_CONTRACT_ID);
 
+const SUPPORTED_WALLET_OPTIONS: IWalletOption[] = [
+  {
+    name: 'mynearwallet',
+    title: 'My Near Wallet',
+  },
+  // support only mynearwallet for now
+  // https://docs.keypom.xyz/docs/next/keypom-sdk/modules#supportedlinkdropclaimpages
+  // {
+  //   title: 'HERE',
+  //   walletName: 'My HERE',
+  //   id: 'herewallet',
+  // },
+];
+
+const DEFAULT_WALLET = SUPPORTED_WALLET_OPTIONS[0];
+
+// used in create drops
+const DEFAULT_TOKEN: IToken = {
+  amount: '',
+  symbol: 'NEAR',
+  wallet: 'near_wallet',
```

Comment:
why is `wallet` a part of the default token, we already have a variable called DEFAULT_WALLET

Reply:
good catch, i think this menu design used to have wallet icon but now its removed, let me remove this property